test/e2e/kubectl/kubectl.go

```diff
@@ -453,6 +449,14 @@ var _ = SIGDescribe("Kubectl client", func() {
 			testableResources := etcd.GetEtcdStorageDataForNamespace(f.Namespace.Name)
 
 			for _, group := range apiGroups {
+				// This significantly limits the scope of this test, but should
+				// ensure it's only covering Kubernetes API resources and not
+				// CRDs or other unexpected resources. For coverage of all
+				// resources in a cluster, remove this line.
+				if !strings.Contains(group.GroupVersion, ".k8s.io") {
```

Comment:
Can you enumerate the known exceptions (existing groups)?

Reply:
Currently that's everything in core, apps, autoscaling, batch, extensions, and policy API groups. On further thought it doesn't seem like it would be that bad to add those to a whitelist here, will push an update shortly.

Reply:
@deads2k Updated this to include all groups that either have `.k8s.io` or do not have `.` in their group name. I _think_ this is a reliable way to exclude CRDs, but I could be missing something.